graphql2/graphqlapp/alert.go

```diff
@@ -177,6 +179,85 @@ func (q *Query) mergeFavorites(ctx context.Context, svcs []string) ([]string, er
 	return svcs, nil
 }
 
+// splitRangeByDuration splits the timeframe given between since and until by the duration provided.
+// Each segment is then transformed into an AlertDataPoint based on the given alerts.
+// The given alerts are required to be sorted by their CreatedAt field.
+func splitRangeByDuration(since, until time.Time, dur timeutil.ISODuration, alerts []alert.Alert) (result []graphql2.AlertDataPoint) {
+	if since.After(until) {
```

Comment:
When I test the query with a `since` date that's after the `until` date the alertMetrics query returns null. I think that makes sense, but I could also see it making sense to returning an empty array. Any preference there?